blueprints/instance-initializer-test/qunit-rfc-232-files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,29 @@
+import Application from '@ember/application';
+import { run } from '@ember/runloop';
+
+import { initialize } from '<%= dasherizedModulePrefix %>/initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import destroyApp from '../../helpers/destroy-app';
+
+module('<%= friendlyTestName %>', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    run(() => {
+      this.application = Application.create();
+      this.application.deferReadiness();
+    });
+  });
+  hooks.afterEach(function() {
+    destroyApp(this.application);
+  });
+
+  // Replace this with your real tests.
+  test('it works', function(assert) {
+    initialize(this.application);
```

Comment:
Instance initializers expect to receive an Ember.ApplicationInstance instance, but this is an Ember.Application instance.

Reply:
To fix that, you would do something like:

```
module(‘asdf’, function(hooks) {

  hooks.beforeEach(function() {
    this.Application = Ember.Application.extend();
    this.application = this.Application.create({ autoboot: true });
    this.application.instanceInitializer({
      name: ‘initializer under test’,
      initialize
    });
  });

  hooks.afterEach(function() {
    runDestroy(this.instance);
    runDestroy(this.application);
  });

  test(‘it works’, async function(assert) {
    this.instance = this.application.buildInstance();
    await this.instance.boot();
    assert.ok(didTheInitializerWork);
  });
});
```

Reply:
@rwjblue Ah yeah overall my bad.  So with this test, `await this.instance.boot();` results in `Assertion Failed: You cannot use the same root element (BODY) multiple times in an Ember.Application`.  Failing on this line - https://github.com/emberjs/ember.js/blob/5f4ac602813d8fbd78f26fc955acf693e0b3d62a/packages/ember-views/lib/system/event_dispatcher.js#L179

Reply:
tested on a 2.17 application.

Reply:
Would you mind pushing up your demo app so I can tinker with it to find the right combo?

Reply:
Also, I made a really bad mistake in my comment above, it should be `{ autoboot: false }` (and I had `true` in the comment)

Reply:
OK, just tested locally, and I think that it will work once updating to `{ autoboot: false }` (sorry about that).

* [example instance-initializer test](https://github.com/rwjblue/__emberjs-PR15945_demo/blob/master/tests/unit/instance-initializers/do-stuff-test.js)
* [diff of changes needed](https://github.com/rwjblue/__emberjs-PR15945_demo/commit/30a36e61691bf4c13ff0321726dc927b9aa93686)